apfloat/src/ieee.rs

```diff
@@ -1971,14 +1971,16 @@ impl<S: Semantics> IeeeFloat<S> {
         //       42039/12655 < L < 28738/8651  [ numerator <= 65536 ]
 
         // Check for MAX_EXP.
-        if normalized_exp.saturating_sub(1).saturating_mul(42039) >= 12655 * S::MAX_EXP as i32 {
+        if normalized_exp.saturating_sub(1).saturating_mul(42039) as i64
+            >= 12655 * S::MAX_EXP as i64
```

Comment:
I forgot the fact that MAX_EXP in octal is 262143 so this multiplication was unmanageable for i32.

Reply:
Thanks for spotting it!